pandas/tests/groupby/test_transform.py

```diff
@@ -346,6 +356,79 @@ def test_transform_transformation_func(transformation_func):
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_corrwith(df_for_transformation_func):
+
+    # GH 27905
+    df = df_for_transformation_func
+    g = df.groupby("A")
+
+    op = lambda x: getattr(x, "corrwith")(df)
+    result = op(g)
+    expected = pd.DataFrame(dict(B=[1, np.nan, np.nan], A=[np.nan] * 3))
+    expected.index = pd.Index([121, 231, 676], name="A")
+    tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_transform_nan(df_for_transformation_func):
+
+    # GH 27905
+    df = df_for_transformation_func
+    g = df.groupby("A")
+
+    df["B"] = [1, np.nan, np.nan, 3, np.nan, 3, 4]
+    result = g.transform("fillna", value=1)
+    expected = pd.DataFrame({"B": [1.0, 1.0, 1.0, 3.0, 1.0, 3.0, 4.0]})
+    tm.assert_frame_equal(result, expected)
+    op = lambda x: getattr(x, "fillna")(1)
+    result = op(g)
+    tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_tshift(df_for_transformation_func):
```

Comment:
should this test be testing `g.transform("tshift", ...)`?

Reply:
The output of this seems to be buggy - ive just raised an issue #32344